### PR TITLE
lab snippets for signs

### PR DESCRIPTION
**Pull request by curstwist**
#### Summary
Content "adds some snippets for future lab content"

#### Purpose of change
Add snippets to improve lab atmosphere.

#### Describe the solution
Begin adding snippets for specific areas of labs.

#### Testing

Play tested.

#### Additional context

I have tried to center some of these in the place sign format.  (it won't be retained when using them as graffiti.).  If anyone knows of a better way to center signs please share it here.

I've begun introducing the use of color for signs as well and included a few samples below.  If we can figure out spacing and some formatting maybe we can even use ascii art.

I hope others come along and expand on the snippets I've begun.  I do plan on adding more categories as I need them for maps.

image
image
image

### CODE REVIEW

File: data/json/snippets/lab_signs.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    "//": "signage for research scientist levels (working lab areas).",
+    "text": [
+      "Keep Equipment Clean and Hygenic",
+      "PPE Required",
+      "<color_yellow_red>Biohazard</color>",
+      "Do not look into laser with remaining eye",
+      "There have been\n[4]\ndays without a life-threatening workplace accident",
+      "Be Alert, Don't get Hurt!",
+      "Safety First\nWork Safely",
+      "<color_white_red>THINK!</color>\n<color_white_red>DON'T CONTAMINATE!</color>\n<color_white>Don't store food in sample refridgerators!</color>",
```
Comment (actual-nh) [inline data/json/snippets/lab_signs.json:51]:
```suggestion
      "<color_white_red>THINK!</color>\n<color_white_red>DON'T CONTAMINATE!</color>\n<color_white>Don't store food in sample refrigerators!</color>",
```

File: data/json/snippets/lab_signs.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      "<color_yellow_red>Biohazard</color>"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "lab_volunteer_snippet",
+    "//": "propaganda signage for the volunteer mutant level.",
+    "text": [
+      "<color_red>For Country</color>\n<color_white>    For</color>\n    <color_cyan>Mom</color>",
+      "<color_white_red>Report any unusual or new symptoms</color>",
```
Comment (actual-nh) [inline data/json/snippets/lab_signs.json:69]: Would they call them symptoms or, say, manifestations?

File: data/json/snippets/lab_signs.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
+[
+  {
+    "type": "snippet",
+    "category": "lab_access_signage",
+    "//": "signage for the civilian areas (public).",
+    "text": [
+      "***LAB***\nAuthorized Personnel Only",
+      "Restricted\nAccess\nArea",
+      "Be Prepared to present ID",
+      "No civilians beyond this point"
```
Comment (actual-nh) [inline data/json/snippets/lab_signs.json:10]: Civilians? Military lab, or what?

File: data/json/snippets/lab_signs.json (hunk 2 above)
Comment (curstwist) [reply data/json/snippets/lab_signs.json:69]: we can do both.  the more variety the better.

Comment (actual-nh) [reply data/json/snippets/lab_signs.json:69]: True. I was thinking that to the subjects, they don't want them to think of illness, so they may not want to call them symptoms. Then again, a _concern_ about symptoms (side effects?) does communicate something positive.

File: data/json/snippets/lab_signs.json [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      "<color_white>Go Boldly Into Your Future!</color>",
+      "<color_white_green>Metamorphosis Checks at 6:00, 12:00, 18:00</color>",
+      "<color_white>Bob palmed his midday dose.\nBob's in the infirmary now\nDon't be Bob: Take doses as advised.</color>",
+      "<color_light_blue>Dream Journals\n   Save\n    Lives</color>",
+      "<color_light_green>Who is your spirit animal?</color>",
+      "<color_yellow>Embrace the Wonder\nEmbrace the Unknown</color>",
+      "<color_red>Your best self is yearning to break free!</color>",
+      "<color_white>Humans only have 5 senses</color>",
+      "<color_yellow_red>Find your spirit animal guide!</color>\n      Group meditation\n      Wednesdays 13:00",
+      "<color_white>  Do Not\n   Share\n<color_white_red>Medications</color>"
```
Comment (cainiaowu) [inline data/json/snippets/lab_signs.json:82]: missing </color> here

**Pull request merged**